src/Tribe/Status/Manager.php

```diff
@@ -359,11 +335,13 @@ public function get_completed_status_by_provider_name( $provider_name ) {
 			$provider_name = get_class( $provider_name );
 		}
 
-		$abbreviated_name = $this->check_for_full_provider_name( $provider_name );
+		$abbreviated_name = $this->get_provider_slug( $provider_name );
 
-		$filtered_statuses = wp_list_filter( $this->statuses[ $abbreviated_name ]->statuses, [
-			'count_completed' => true,
-		] );
+		$filtered_statuses = wp_list_filter(
+			$this->statuses[ $abbreviated_name ]->statuses, [
```

Comment:
:no_entry_sign: Only one argument is allowed per line in a multi-line function call

<sub>hash: 0c154b015c9d5be3903d277978562148</sub>

Reply:
fixed